github/ci/prow-deploy/vars/kubevirtci-testing/secrets.yml

```diff
@@ -37,3 +37,5 @@ windowsProductKeys:
   sysprep-001: "AAAAA-BBBBB-CCCCC-DDDDD-EEEEE"
 
 fossaToken: d3c481176ace1b6c5eb417b0d3dd01497
+
+prowKubevirtbotSSHPrivateKey: "fakie fake"
```

Comment:
@dhiller When adding a new variable for Ansible, how would I know if I need to add it for all the roles or for a particular role?